crates/sqlexec/src/planner/physical_plan/copy_to.rs

```diff
@@ -37,7 +93,11 @@ impl ExecutionPlan for CopyToExec {
     }
 
     fn schema(&self) -> Arc<Schema> {
-        GENERIC_OPERATION_AND_COUNT_PHYSICAL_SCHEMA.clone()
+        if self.is_dummy_query() {
+            self.source.schema()
+        } else {
+            GENERIC_OPERATION_AND_COUNT_PHYSICAL_SCHEMA.clone()
+        }
```

Comment:
I believe this should always be the `GENERIC_...` schema as that's what we use to determine that the operation was a COPY.

Reply:
Yeah! I'll experiment a bit here. I want to wrap the source query's `SendRecvJoinExec` inside a `CopyToLocal` exec. Because we're using extensions, DF checks that the logical and physical plans have the same schema. We eventually get the same schema, but the DDL planner does not handle it. I'll continue working here once I'm done with values exec.